LocationServices/LocationServices/Scenes/Tracking/ViewModel/TrackingViewModel.swift

```diff
@@ -19,6 +19,7 @@ struct RouteStatus {
     var busAnnotation: ImageAnnotation?
     var routeCoordinates: [RouteCoordinate] = []
     var geofenceIndex = 1
+    var timer: Timer? = nil
```

Comment:
Where is this used? Is that in a follow up PR? If not, could you please clean this up?